pkg/api/v1beta1/types.go

```diff
@@ -666,8 +684,6 @@ type ResourceList map[ResourceName]util.IntOrString
 // The name of the minion according to etcd is in ID.
 type Minion struct {
 	TypeMeta `json:",inline"`
-	// Queried from cloud provider, if available.
-	HostIP string `json:"hostIP,omitempty" description:"IP address of the node"`
```

Comment:
You'll need to do something like what we did for resources. v1beta1 clients may expect HostIP to be populated. We'll need to pick an address/hostname to use to populate it if it's empty but the new list is non-empty.